timon/scripts/isup.py

```diff
@@ -3,18 +3,31 @@
 import sys
 
 import requests
+from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.exceptions import InsecureRequestWarning
+from requests.packages.urllib3.util.retry import Retry
 
 from .flags import FLAG_MAP
 
 requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
 
 
+RETRY_STRATEGY = Retry(
+    total=3,
```

Comment:
suggest to increase `total`, and to reduce `backoff_factor` and `timeout` e.g. `total = 18` `backoff_factor=10` timeout=`9`

should see lateron how to pass this params via the timon config

Reply:
Euhhh, the retry time is exponential, so a total of 18 retries is too much. With a backoff_factor at 10, retry times in seconds will be: 5, 10, 20, 40, 80, 160, 320, 640, 1280, 2560, ... With 11 retries it reach more than 1 hour, i don't think it's a good idea

Reply:
as far as I know you can decide if the time is exponential or linear. I think in our case we should go for linear.
We cannot have one probe blocking for too long I think. If not we should do this one level higher in timon.

Reply:
the isup probe is normally called every two minutes to have it block for a longer time is kind of strange. We can discuss this on monday